python/examples/integrations/flask_streaming/flask_with_whylogs.ipynb

```diff
@@ -0,0 +1,836 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Integrating Whylogs into your Flask Flow\n",
+    "\n",
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/whylabs/whylogs/blob/mainline/python/examples/integrations/flask_with_whylogs.ipynb)\n",
+    "\n",
+    "Now that you've gone through the basics, let's get into how to integrate whylogs into your current work flow. Today, let's look at Flask to build a web app that wil get data from IRIS and us it for a prediction. Notice that we will log both the imput received and the prediction! "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "First let's get our environment ready"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install -q pandas utils joblib scikit-learn 'whylogs[whylabs]'"
```

Comment:
maybe we could install the other packages, but let's make the `pip install whylogs[whylabs]` commented, so we don't install this every time we run the test on it (even though this will probably be skipped, since we need Flask, docker, etc.)

Reply:
Deal!